bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/LabelImpl.java

```diff
@@ -78,6 +78,10 @@ public Boolean isVisible() {
      */
     @Override
     public String getValue() {
-        return ComponentUtils.translate(properties.get(PN_TITLE, this.defaultTitle), PN_TITLE, properties, i18n);
+        if (properties.containsKey(PN_TITLE))
```

Comment:
If PN_TITLE is not present in properties, we always return the default title, which is the name of the text box. This behavior is as per AF v1. Why do we need to change this ?

Reply:
@rismehta I see in AF1 we have the behavior as the component's title remains blank if given blank. We are trying to implement the same here. Not sure if i am missing anything. Pls suggest..??
![image](https://github.com/adobe/aem-core-forms-components/assets/112863330/96ba3809-e50f-449c-939e-92890c8fd8f5)